### DISCUSSION TITLE
Workflow group 2 review

### DISCUSSION THREAD

**Discussion by MichaelMugaiga** [HYF-Class19/workflow-group-2#15] [General]
Can we discuss the issues we are still facing with our workflow assignement?

**Comment by sammou00** [comment]
@MichaelMugaiga  What issues do you have ?

**Reply by MichaelMugaiga** [reply]
I don't have any issues now. maybe my colleagues

**Comment by sammou00** [comment]
Please Note  : 
- Don't add your PR to the project board only issues , as we agreed in the class.

**Comment by sammou00** [comment]
Hello, Good job, I think adding your images  to your md files is enough , why add them  to  README.md  ?

**Reply by MichaelMugaiga** [reply]
thank you, i had only put a link in the README.md file to my md but it seems John and Anita want the picture like this. but i will suggest to them that we remove pictures once all group members have added their profiles. Thanks

**Reply by sammou00** [reply]
It is just a suggestion, there is no right or wrong. Please move your issues to `done` when your colleagues done reviewing them.
